state/protocol/badger/state.go

```diff
@@ -599,3 +609,9 @@ func (state *State) updateCommittedEpochFinalView(snap protocol.Snapshot) error
 
 	return nil
 }
+
+func (m *State) isEpochEmergencyFallbackTriggered() (bool, error) {
+	var triggered bool
```

Comment:
EECC triggered or not is only checked on a finalized block, that means it won't trigger again on other forks, it should only be triggered once. 

I wonder if it's useful to store a block ID or block height of the block that triggered EECC. Would it provide more context than a bool?

Reply:
👍  good idea - will add this

Reply:
https://github.com/onflow/flow-go/pull/1569/commits/5a5b7e0ae9115b059b3f1a9714397c9fb0593674